images/bazel/cloudbuild.yaml

```diff
@@ -2,23 +2,23 @@ steps:
   - name: gcr.io/cloud-builders/docker
     args: 
     - build
-    - --tag=gcr.io/$PROJECT_ID/launcher.gcr.io/google/bazel:$_NEW_VERSION-from-$_OLD_VERSION
+    - --tag=gcr.io/$PROJECT_ID/launcher.gcr.io/google/bazel:$_GIT_TAG-$_CONFIG
     - --build-arg=NEW_VERSION=$_NEW_VERSION
     - --build-arg=OLD_VERSION=$_OLD_VERSION
-    - --build-arg=IMAGE_ARG=gcr.io/$PROJECT_ID/launcher.gcr.io/google/bazel:$_NEW_VERSION-from-$_OLD_VERSION
+    - --build-arg=IMAGE_ARG=gcr.io/$PROJECT_ID/launcher.gcr.io/google/bazel:$_GIT_TAG-$_CONFIG
     - .
+
   - name: gcr.io/cloud-builders/docker
```

Comment:
Could probably override entry point to a shell and run N tag commands?

Reply:
Not actually sure if cloudbuild allows that? But nifty idea. Let me investigate later.

Reply:
It does -- 
https://cloud.google.com/cloud-build/docs/build-config#entrypoint